src/components/CountriesList.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { FaChevronLeft } from 'react-icons/fa';
+
+import WORLD_MAP from '../assets/img/world-map.png';
+import getCurrentDate from '../utils/currentDate';
+
+import { fetchCovidStats } from '../store/covidStats';
+import Country from './Country';
+import CustomDatePicker from './CustomDatePicker';
+
+const CountriesList = () => {
+  const currentDate = getCurrentDate();
+  const dispatch = useDispatch();
+  const { casesByCountry, totalCases } = useSelector((state) => state);
```

Comment:
- [ ] Your code is working correctly however for this project you need to add a feature that will filter the received data from the appear and filter it by some category. for example the country continent or the country name. so the user can for example only view countries from a certain continent or any other property